### PR TITLE
Updated zookeeper dependency and edited kafka properties to be schema…

### PR DESCRIPTION
**Pull request by ataylor4**
...and format on release branch

https://github.com/caskdata/cdap/pull/3356 without front-end changes and branched from release

### CODE REVIEW

**Comment by gokulavasan** [comment]
LGTM :+1:

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/KafkaSource.java [hunk 1]
```diff
@@ -65,8 +65,8 @@
   public static final String KAFKA_ZOOKEEPER = "kafka.zookeeper";
   public static final String KAFKA_BROKERS = "kafka.brokers";
   public static final String KAFKA_DEFAULT_OFFSET = "kafka.default.offset";
-  public static final String KAFKA_SCHEMA = "kafka.schema";
-  public static final String KAFKA_FORMAT = "kafka.format";
+  public static final String KAFKA_SCHEMA = "schema";
```
Comment (hsaputra) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/KafkaSource.java]: If the constant value changed, then it is preferable to also change the variable name to match it

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/KafkaSource.java [hunk 2]
```diff
@@ -65,8 +65,8 @@
...
+  public static final String KAFKA_FORMAT = "format";
```
Comment (hsaputra) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/realtime/source/KafkaSource.java]: If the constant value changed, then it is preferable to also change the variable name to match it

**Comment by hsaputra** [comment]
Small nit on variable name, other than that LGTM

**Pull request merged**